### PR TITLE
Convert trend depth toggle to functional component

### PR DESCRIPTION
**Pull request by flacoman91**
updated code, tests and package.json coverage thresholds.

I went ahead and did this since it already has been done in the massive RTK pull request.  I didn't want you to have to reinvent the wheel.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -135,10 +135,10 @@
     ],
     "coverageThreshold": {
       "global": {
-        "branches": 79,
-        "functions": 86,
-        "lines": 86,
-        "statements": 86
+        "branches": 80,
```
Comment (cdmh219) [inline package.json:138]: Yeah I think this can be moved back to its original values

File: src/components/Trends/TrendDepthToggle.js [hunk 2]
```diff
@@ -14,63 +18,90 @@ const lensMap = {
...
+  const diff = totalResultsLength - resultCount;
+  const hasToggle = showToggle(lens, focus, totalResultsLength, queryCount);
+
+  // hide on Overview and Focus pages
+  if (focus || lens === 'Overview') {
+    return null;
+  }
+
+  if (hasToggle) {
+    if (_showMore(queryCount, resultCount)) {
```
Comment (cdmh219) [inline src/components/Trends/TrendDepthToggle.js:65]: Not a big deal, but would the underscore at the start of `_hasMore` be necessary here? I assumed they were present as a way to indicate they were on the instance of the TrendDepthToggle class component.

File: src/components/Trends/TrendsPanel.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ import Select from '../RefineBar/Select';
 import { Separator } from '../RefineBar/Separator';
 import StackedAreaChart from '../Charts/StackedAreaChart';
 import { TabbedNavigation } from '../TabbedNavigation';
-import TrendDepthToggle from './TrendDepthToggle';
+import { TrendDepthToggle } from './TrendDepthToggle';
```
Comment (cdmh219) [inline src/components/Trends/TrendsPanel.js:29]: Is there a standard around how to export these components? I've been doing `export default`, which causes it to import like the removed version of line 29, but also noticed it being exported as a variable.

File: src/components/Trends/TrendDepthToggle.js (hunk 2 above)
Comment (flacoman91) [reply src/components/Trends/TrendDepthToggle.js:65]: No.  I think someone else did the underscore for functions in the past as private functions.  

Normally, I would not use them.  I'll remove them.

File: src/components/Trends/TrendsPanel.js (hunk 3 above)
Comment (flacoman91) [reply src/components/Trends/TrendsPanel.js:29]: I prefer named exports with the brackets and I think we should stick to one convention.  Looks like there's also benefits to being explicit with the named exports.


https://react.dev/learn/importing-and-exporting-components#default-vs-named-exports
https://dev.to/phuocng/avoid-using-default-exports-a1c#:~:text=Default%20exports%20are%20used%20to,a%20function%20or%20a%20class.

We should see if we can find an eslint rule to enforce this standard.

Comment (flacoman91) [reply src/components/Trends/TrendsPanel.js:29]: Also with the named imports, you can have more than one export per file and you can unit test them more easily.

**Review by cdmh219 — APPROVED** [review]

**Pull request merged**